t/data/04-complex_template.html

```diff
@@ -651,4 +651,4 @@
 </div>
 <?lsmb END ?>
 </body>
-</html>
+<?lsmb end_html ?>
```

Comment:
hmm. This one belongs in the previous PR I merged, I think. Well, lets leave it that way.

Reply:
Missed 1. Sorry